sqlglot/dialects/starrocks.py

```diff
@@ -31,6 +45,65 @@ class Parser(MySQL.Parser):
             "REGEXP": exp.RegexpLike.from_arg_list,
         }
 
+        PROPERTY_PARSERS = {
+            **MySQL.Parser.PROPERTY_PARSERS,
+            "DISTRIBUTED": lambda self: self._parse_distributed_property(),
+            "DUPLICATE": lambda self: self._parse_duplicate(),
+            "PROPERTIES": lambda self: self._parse_wrapped_properties(),
```

Comment:
Let's move these to the base parser instead, since we already did this for the corresponding generator methods.

Reply:
OK.

`PROPERTIES` on line 52 is not suitable for moving, because it may affect other dialects. The following test will fail:
```
self.validate_identity(
"ALTER TABLE people SET PROPERTIES x = 'y'", check_command_warning=True
)

Reply:
Got it